### ISSUE TITLE
Priority must be given to control plane STX transactions

### ISSUE THREAD

**Issue by jferrant** [stacks-network/stacks-core#3981]
To ensure that DKG, sBTC ops, tenure changes, etc. do not get lost in the plethora of other user transactions, we need to determine a way to give priority/fast lane pass to control plane transactions.

This will involve changes to both the miner/signer logic.

- [x] https://github.com/stacks-network/stacks-core/issues/4243
- [x] #4273

**Comment by jcnelson** [comment]
We will want to be better informed about the StackerDB performance in order to find the best way to implement this.

**Comment by saralab** [comment]
To do:
Create 2 different tasks 

These subtasks can be in parallel and separate PRs

**Comment by saralab** [comment]
Research Discussion: 
How do miners receive information from Stackers?  This issue is blocked by the discussion. 

- How do miners receive tenurechange txs: Listen to every event , will not be a huge burden for the miners. 
- How do miners prioritize within those block building : 
- How does the stacker set become fault tolerant to losing n/w connection 
- Handling the votes for the prepare phase should be a separate issue

**Comment by saralab** [comment]
Relevant to sBTC , not needed for Nakamoto

**Comment by saralab** [comment]
Implemented as part of https://github.com/stacks-network/stacks-core/pull/4287
